public/src/components/channelManagement/bannerTests/bannerUiSelector.tsx

```diff
@@ -0,0 +1,189 @@
+import React, { useEffect, useState } from 'react';
+import {
+  FormControl,
+  FormControlLabel,
+  FormLabel,
+  MenuItem,
+  Radio,
+  RadioGroup,
+  Select,
+} from '@material-ui/core';
+import {
+  BannerDesignName,
+  BannerTemplate,
+  BannerUi,
+  isBannerTemplate,
+  uiIsDesign,
+} from '../../../models/banner';
+import { BannerDesign } from '../../../models/BannerDesign';
+import { shouldShowBannerDesignsFeature } from '../../../utils/features';
+
+interface BannerUiSelectorProps {
+  ui: BannerUi;
+  onUiChange: (updatedUi: BannerUi) => void;
+  editMode: boolean;
+  designs: BannerDesign[];
+}
+
+const templatesWithLabels = [
+  { template: BannerTemplate.AusAnniversaryBanner, label: 'Aus 10 yr moment' },
```

Comment:
hopefully this work will eventually lead to this type of config being removed!

Reply:
Yes! (I've not added this btw, it's showing up as new because of the file rename).